src/DatabaseMetadata/ToArrayInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorCommon\DatabaseMetadata;
+
+interface ToArrayInterface
```

Comment:
Nevim, jestli mi to nepřijde zbytečný. Samo o sobě mi to nepřijde použitelný. Typicky nebudeš hintovat na ToArrayInterface - spíš na ten value object...